README.md

```diff
@@ -235,58 +235,69 @@ For headaches, take Ibuprofen
+module mode) with the additional constraint that the list of packages must be
+main packages. Each argument takes the form $main_pkg[@$version].
 
 By default, gobin is said to operate in global mode. If the -m flag is provided
 then it is said to operate in main-module mode, where the path to the main
 module's go.mod is given by go env GOMOD.
 
-The version "latest" matches the latest available tagged version. If no version
-is specified, gobin behaves differently in global and main-module modes. In
-global mode, gobin attempts to resolve the latest version available in the
-module download cache. In main-module module, gobin attempts to resolve the
-current version via the main module's go.mod. If this resolution fails in
-either mode, "latest" is assumed and gobin resolves via the network.
+The version "latest" matches the latest available tagged version for the module
+containing the main package. gobin will make a network request to resolve
+"latest" unless -nonet is provided, in which case it will attempt to resolve
+"latest" within the GOPATH module download cache. Versions that take the form
+of a revision identifier, e.g. a branch name, can only be resolved with a
+network request and hence are incompatible with -nonet.
 
-In global mode, gobin installs the main packages to the directories
-gobin/$module@$version/$main_pkg under your user cache directory. See the
-documentation for os.UserCacheDir for OS-specific details on how to configure
-its location.
-
-In main-module mode, gobin installs the main packages to the directories
-.gobincache/$module@$version/$main_pkg under the directory containing the main
-module's go.mod.
+If no version is specified for a main package, gobin behaves differently in
+global and main-module modes. In global mode, gobin attempts to resolve the
+latest tagged version available in the module download cache. In main-module
+module, gobin attempts to resolve the current version via the main module's
+go.mod. If this resolution fails in either mode, "latest" is assumed as the
+version.
 
 By default, gobin installs the main packages to $GOBIN (or $GOPATH/bin if GOBIN
 is not set, which defaults to $HOME/go/bin if GOPATH is not set).
 
-The -r flag takes exactly one main package argument and runs that package.  It
-is similar therefore to go run. Any arguments after the single main package
+The -run flag takes exactly one main package argument and runs that package.
+It is similar therefore to go run. Any arguments after the single main package
 will be passed to the main package as command line arguments.
 
 The -p flag prints the cache install path for each of the provided packages
 once versions have been resolved.
 
-The -r and -p flags are mutually exclusive.
+The -run and -p flags are mutually exclusive.
 
-The -g flag forces gobin to perform a network fetch for the provided main
-packages.
+The -u flag forces gobin to check for the latest tagged version. As such, it
```

Comment:
I know what you're trying to say, but this sounds like it contradicts itself. How about:

     The -u flag forces gobin to check for the latest tagged version. As such, it
     has no effect if an exact version (v1.2.3, for example) is specified, because
     that will always resolve to the same thing.

?

Reply:
This was more trying to capture the revision identifier case, e.g. a branch name